src/elements/content-preview/preview-header/PreviewHeader.scss

```diff
@@ -5,11 +5,21 @@
     flex: 0 0 60px;
     align-items: center;
     justify-content: space-between;
-    padding: 0 15px;
+    padding: 0;
     font-size: 13px;
     background-color: $bdl-gray-02;
     border-bottom: 1px solid $bdl-gray-10;
 
+    // Box Annotations selector
+    .bp-header {
+        flex: 1;
+        height: 60px;
+    }
+
+    .bp-base-header {
```

Comment:
This isn't a flex container like its parent, which seems like it will cause some visual issues. Can you test this with the OpenWith button and post screenshots of before/after?